wasmtime-runtime/src/traphandlers.rs

```diff
@@ -18,9 +18,16 @@ use std::vec::Vec;
 extern "C" {
     fn setjmp(env: *mut jmp_buf) -> c_int;
     fn longjmp(env: *const jmp_buf, val: c_int) -> !;
+    #[cfg(target_os = "windows")]
+    fn _resetstkoflw() -> c_int;
+}
+
+enum PostUnwindAction {
+    FixStack,
 }
 
 thread_local! {
+    static POST_UNWIND_ACTIONS: RefCell<Vec<PostUnwindAction>> = RefCell::new(Vec::new());
```

Comment:
Instead of being `RefCell<Vec<T>>`, could this perhaps just be `Cell<bool>`? When this is pushed to, it's immediately consumed with an exceptional return, right?

Reply:
`Cell<bool>` is enough. I just came with more general solution, but yeah, it's needlessly complicated now. I'll fix it.